google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/OAuth2Utils.java

```diff
@@ -21,18 +21,34 @@
 import com.google.api.client.http.HttpTransport;
 
 import java.io.IOException;
+import java.net.SocketTimeoutException;
 import java.nio.charset.Charset;
 import java.util.Collection;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 /**
  * Utilities used by the com.google.api.client.googleapis.auth.oauth2 namespace.
- *
  */
-class OAuth2Utils {
+public class OAuth2Utils {
```

Comment:
Are you expecting the application to use this directly? Should it be marked as Beta or something indicating API-stability guarantees?